editor/plugins/script_text_editor.h

```diff
@@ -109,6 +112,14 @@ class ScriptTextEditor : public ScriptEditorBase {
 	void _goto_line(int p_line) { goto_line(p_line); }
 	void _lookup_symbol(const String &p_symbol, int p_row, int p_column);
 
+	enum CONVERT_CASE
```

Comment:
The convention for enum scopes is CamelCase, and the brackets open on the same line, so it should be `enum ConvertCase {`.